charts/k8ssandra/values.yaml

```diff
@@ -97,6 +97,10 @@ cassandra:
     # environments.
     size: 1
 
+    # -- Number of tokens within the datacenter. If not defined, the default values will be
+    # used. 
+    # num_tokens: 16
```

Comment:
I am a bit torn on the property name. Thinking out loud here...

On the one hand we are for the most part using camel case, so to be consistent it ought to be `numTokens`. On the other hand the fact that the property name is the same as it is in cassandra.yaml makes it crystal clear what the property is. 

I say leave it as `num_tokens`. If we are adding a property that maps directly to a property in cassandra.yaml, then using the same name adds clarity.